zetaclient/keys_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 	hd "github.com/cosmos/cosmos-sdk/crypto/hd"
-	cKeys "github.com/cosmos/cosmos-sdk/crypto/keyring"
+	cryptokeyring "github.com/cosmos/cosmos-sdk/crypto/keyring"
```

Comment:
Or simply `keyring` would also work

Reply:
At line 11 in this file its already there "github.com/99designs/keyring".